docs/webhooks.rst

```diff
@@ -61,13 +62,13 @@ Bitbucket
 * Finish by clicking **Save**
 
 GitLab
-~~~~~~
+------
 
 * Go to the **Settings** page for your project
 * Click **Integrations**
 * For **URL**, use the URL of the integration on Read the Docs, found on the
   :ref:`integration detail page <integration-detail>` page
-* Leave the default **Push events** selected
+* Leave the default **Push events** selected and mark **Tag push events** also
```

Comment:
Not sure, but I think we need the _Tag push events_ here also...

Reply:
Seems like we'd want them. +1